public/app/plugins/datasource/loki/querybuilder/operations.ts

```diff
@@ -238,6 +238,28 @@ export function getOperationDefinitions(): QueryBuilderOperationDef[] {
       addOperationHandler: addLokiOperation,
       explainHandler: (op) => `Return log lines that contain string \`${op.params[0]}\`.`,
     },
+    {
+      id: LokiOperationId.LineContainsInsensitive,
+      name: 'Line contains (case insensitive)',
```

Comment:
I'd name this just `Line contains case insensitive`. I haven't seen any other operation that uses brackets, so I wouldn't introduce new pattern like this.